test/integration/targets/inventory_gcp_conformance/runme.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# set the output dir
+if [ -z "${OUTPUT_DIR+null}" ]; then
+    export OUTPUT_DIR=${PWD}
+fi
+
+# Create fake credentials
+cat << EOF > "$OUTPUT_DIR/gcp_credentials.json"
+{
+  "type": "service_account",
+  "project_id": "foo",
+  "private_key_id": "FOO",
+  "private_key": "BAR",
+  "client_email": "FOOBAR@foo.iam.gserviceaccount.com",
+  "client_id": "BAZ",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "MEH"
+}
+
+EOF
+
+#################################################
+#   RUN THE SCRIPT
+#################################################
+
+# run the script first
+cat << EOF > "$OUTPUT_DIR/gce.ini"
+[gce]
+gce_service_account_pem_file_path = $OUTPUT_DIR/gcp_credentials.json
+gce_service_account_email_address = FOOBAR@foo.iam.gserviceaccount.com
+gce_project_id = foo
+gce_zone = us-east1-d
+
+EOF
+
+rm -f script.out
+./gce.sh --list | tee "$OUTPUT_DIR/script.out"
+RC=$?
+if [[ $RC != 0 ]]; then
+    exit $RC
+fi
+rm -f "$OUTPUT_DIR/gce.py"
+rm -f "$OUTPUT_DIR/gce.ini"
+
+#################################################
+#   RUN THE PLUGIN
+#################################################
+
+# run the plugin second
+cat << EOF > "$OUTPUT_DIR/test.gcp.yml"
+plugin: gcp_compute
+projects:
+  - foo
+auth_kind: serviceaccount
+service_account_file: $OUTPUT_DIR/gcp_credentials.json
+compose:
+  ansible_ssh_host: networkInterfaces[0].accessConfigs[0].natIP
```

Comment:
The user errors of dot notation don't pertain to these conformance tests either, but you mentioned preferring not to use dot notation, so that should probably be fixed here and below.

Reply:
Yeah, I copied over an older iteration of the one from the Tower PR and forgot to "fix" it. Thanks :)